recipes/acpype/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "acpype" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+    url: https://github.com/alanwilter/{{ name }}/archive/d99be8e17e3f999e06a401e5efdbe5f2fc9a2959.tar.gz
+    sha256: 43736873feaa791c76ee02a6eae6aba5fb661149a915d8772ec5a4a1f920473d
+build:
+    number: 0
+    script:
+        - mv recipe/setup_conda.py setup.py
+        - "{{ PYTHON }} -m pip install . -vv"
+    noarch: python
+    entry_points: 
+        - acpype=anaconda.acpype:init_main
+requirements:
+    host:
+        - python >=3
+        - pip
+    run:
+        - openbabel
+        - python >=3
+        - pip
+        - ambertools
```

Comment:
This does not exist on win or osx. So you need to skip those builds here as well.

Reply:
If it's `noarch` it doesn't matter.

The feedstock will only build on Linux and attempts to install it will result in an error like "ambertools is not available for the platform osx64".

Reply:
Oh, it is noarch, that's true. Thanks for the merge.